tests/Neo.UnitTests/SmartContract/UT_InteropService.NEO.cs

```diff
@@ -121,12 +121,14 @@ public void TestContract_Create()
 
             var script_exceedMaxLength = new NefFile()
             {
-                Script = new byte[NefFile.MaxScriptLength - 1],
+                Script = new byte[ExecutionEngineLimits.Default.MaxItemSize - 50],
                 Source = string.Empty,
                 Compiler = "",
-                Tokens = System.Array.Empty<MethodToken>()
+                Tokens = Array.Empty<MethodToken>()
             };
-            script_exceedMaxLength.CheckSum = NefFile.ComputeChecksum(nef);
```

Comment:
Note: This checksum was incorrectly computed

Reply:
What this means, you have corrected the checksum? will this cause inconstancy?

Reply:
No, is just a UT but it was wrong